core/src/test/scala/pickling/run/runtime-instantiation-static-picklers.scala

```diff
@@ -0,0 +1,219 @@
+package scala.pickling.runtime
+
+import org.scalatest.FunSuite
+import scala.pickling._, scala.pickling.Defaults._, json._
+
+case class Container[T](value: T)
+
+object RuntimeHelper {
+  /* Instantiate a class for an already generated `Unpickler` */
+  def getInstance[T](className: String): T = {
+    try {
+      val clazz = Class.forName(className)
+      println(s"Create instance of $clazz")
+      clazz.newInstance().asInstanceOf[T]
+    } catch {
+      case ex: Throwable =>
+        scala.concurrent.util.Unsafe.instance.allocateInstance(
+          Class.forName(className)
+        ).asInstanceOf[T]
+    }
+  }
+}
+
+class StaticJavaRuntime extends FunSuite {
+
+  implicit val staticOnly = static.StaticOnly
+
+  test("runtime instantiated pickler should work") {
+
+    val c = Container(List(1,2,3,4,5))
+    val containerPickler = implicitly[Pickler[Container[List[Int]]]]
+    val pickled = c.pickle.value
+
+    val picklerClassName = containerPickler.getClass.getName
+    val runtimePickler = RuntimeHelper.getInstance[Pickler[Container[Any]]](picklerClassName)
```

Comment:
BTW, I'm also doing this because I want to avoid using runtime pickling, I only want static picklers. As I'm sharing the jars between the server and the client, there's no problem.